ci/experimental/docker/connector_build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# We use manylinux1 base image because pyarrow_manylinux2010 has a bug and wheel failed to be audited
```

Comment:
I think this file could go straight into `ci/docker/connector_build_aarch/Dockerfile`

Reply:
I think this comment isn't true anymore
```suggestion
```